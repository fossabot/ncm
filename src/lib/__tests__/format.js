import { objToDotenv } from '../format';

describe('objToDotenv', () => {
  it('transform json into dotenv style key-value content', () => {
    const obj = {
      id: 1,
      foo: 'bar',
    };
    const result = objToDotenv(obj);
    expect(result).toMatchSnapshot();
  });
});
