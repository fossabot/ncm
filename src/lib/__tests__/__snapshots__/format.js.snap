// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`objToDotenv transform json into dotenv style key-value content 1`] = `
"id=1
foo=bar
"
`;
